t_http2/test_h2_sticky_scheduler.py

```diff
@@ -0,0 +1,164 @@
+"""Test module for http2 and Sticky Cookie Scheduler."""
```

Comment:
Why do we need a separate directory for this only test? Is there a reason this can't be placed in, say, `./sessions/test_sessions.py`?

Reply:
I had plans to place tests in directories with prefix, but for now I have no time because main project.
I moved file to dir `/sessions/`.